docs/flow/reference/components/discovery.relabel.md

```diff
@@ -6,15 +6,27 @@ title: discovery.relabel
 
 # discovery.relabel
 
+In Flow, targets are defined as sets of key-value pairs called _labels_.
+
 `discovery.relabel` rewrites the label set of the input targets by applying one
 or more `relabel_config` steps. If no relabeling steps are defined, then the
-input targets will be exported as-is.
+input targets are exported as-is.
 
 The most common use of `discovery.relabel` is to filter Prometheus targets or
```

Comment:
Do we want to drop the mention of Prometheus here? These targets will be used in a lot of places, even if the `prometheus.*` components are the only real use case for them at the moment.

We could just say "filter targets" instead.